src/components/Layout/DrawerNavigation.tsx

```diff
@@ -118,18 +118,30 @@ export const DrawerNavigationComponent: FunctionComponent<
   const onPressCloseDrawer = (): void => {
     navigation.dispatch(DrawerActions.closeDrawer());
   };
-  const releaseChannel: string | undefined = Constants.manifest?.releaseChannel;
+  const branchName: string | undefined = (Constants.manifest2?.metadata as any)
+    ?.branchName
+    ? Constants.manifest?.releaseChannel
+    : (Constants.manifest2?.metadata as any)?.branchName;
+  const manifestVersion = Constants.manifest?.version
```

Comment:
expo sdk45 stores env var in 2 separate places depending on whether you're opening a build or an update. sdk46 will combine back into 1